src/pagination/pagination.spec.ts

```diff
@@ -519,6 +522,22 @@ describe('ngb-pagination', () => {
       expectPages(fixture.nativeElement, ['« Previous', '1', '+2', '-» Next']);
     });
 
+    it('should render and respond to direction links changes', () => {
+      const html = '<ngb-pagination [collectionSize]="collectionSize" [page]="1" [firstText]="firstText"' +
+          ' [previousText]="previousText" [nextText]="nextText" [lastText]="lastText"' +
+          ' [boundaryLinks]="boundaryLinks"></ngb-pagination>';
```

Comment:
Got it. Thank you - I see that being used too in other templates in the spec file. I will do a commit tomorrow for the template string change and the change for the more detailed description for the public properties. 

Given the additional context I provided would you still like me to proceed with applying the other 2 changes you have suggested (getting rid of the private properties and combining the non-SR and SR properties into one set) ?